app/src/testDebug/java/io/reark/rxgithubapp/utils/StethoInstrumentationTest.java

```diff
@@ -76,6 +76,5 @@ public void testDecorateNetwork() {
 
         instrumentation.decorateNetwork(okHttpClient);
 
-        verify(instrumentation).addInterceptor(eq(okHttpClient), eq(interceptor));
```

Comment:
This verify doesn't work, so Interceptor it is immutable list.
https://square.github.io/okhttp/3.x/okhttp/okhttp3/class-use/Interceptor.html